app/models/foreman_azure_rm/azure_rm.rb

```diff
@@ -215,7 +215,7 @@ def create_vm(args = {})
       end
       vm                       = client.create_managed_virtual_machine(
           name:                            args[:vm_name],
-          location:                        args[:location],
+          region:                          args[:region],
```

Comment:
This is fog's interface [property](https://github.com/fog/fog-azure-rm/blob/master/lib/fog/azurerm/requests/compute/create_virtual_machine.rb#L25), it should remain `location`